infra-as-code/bicep/modules/hubNetworking/parameters/hubNetworking.parameters.min.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
Is this really a `min`? is there no trimming we can do?

Reply:
min trims the following from the file:

- parLocation
- parCompanyPrefix
- parHubNetworkName
- parBastionName
- parDdosPlanName
- parAzureFirewallName
- parFirewallPoliciesName
- parHubRouteTableName
- parTags

I thought we had discussed and agreed as a team that all functionality changing parameters would remain in the `.min.json`? Maybe I misinterpreted?